requirements.txt

```diff
@@ -0,0 +1,42 @@
+# Sphinx dependencies
+Sphinx>=7.0.0
+sphinx_rtd_theme==2.0.0
+
+ase>=3.18.0
+contourpy>=1.2.0
+coverage>=7.5.0
+cycler>=0.12.0
+exceptiongroup>=1.2.0
+fonttools>=4.53.0
+importlib_resources>=6.4.0
+iniconfig>=2.0.0
+kiwisolver>=1.4.5
+matplotlib>=3.9.0
+
+metatensor==0.2.0
+metatensor-core==0.1.8
+metatensor-learn==0.2.2
+metatensor-operations==0.2.2
+
+numpy>=1.13.0
+packaging>=24.0
+pillow>=10.0.0
+pluggy>=1.5.0
+pyparsing>=3.1.0
+pytest>=8.2.0
+python-dateutil>=2.9.0.post0
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-manylinux_2_17_x86_64.manylinux2014_x86_64.whl; sys_platform=="linux"
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-macosx_11_0_arm64.whl; sys_platform=="darwin" and platform_machine=="arm64"
+
+rascaline @ https://luthaf.fr/nightly-wheels/rascaline/rascaline-0.1.0.dev558-py3-none-macosx_10_9_x86_64.whl; sys_platform=="darwin" and platform_machine=="x86_64"
```

Comment:
I would probably suggest not pinning this to a specific platform or version -- does it not build from the github anymore?

Reply:
I was running into issues with rascaline, I think due to the compilation process, so I set it to install from the pre-built wheels instead.  I will double check this, though.